README.md

```diff
@@ -4,6 +4,46 @@ Portable Linux (i.e. a generic Linux version) packaging for [ungoogled-chromium]
 
 Portable Linux builds can run on **any Linux distribution** (that regular Chromium supports).
 
+Portable Linux binary can be installed, alternatively:
+
+```
+$ cd ~/Downloads
+$ wget https://github.com/Eloston/ungoogled-chromium-binaries/releases/download/75.0.3770.80-1.2/ungoogled-chromium_75.0.3770.80-1.2_linux.tar.xz
```

Comment:
This is tied to a specific version, and it's in the introduction section of this document. Should it be moved?

Reply:
Agreed on all three counts:-

1. It would be better if moved out of the introductory section.

2. It could be moved, say, after the section on building from source and just before the last section ("Developer info"), perhaps into a new section titled "Installing Compiled Binary". However, please call the section and move it (if that is possible) however you think is best and amend any title/wording as you wish, or I will happily try to move/edit this to that location.  

3. All references to a specific version can be removed, say, by using the following introductory paragraph.  Note:  The original guidance had references to a specific version because I suspect that packages that may be able to be installed by the general population (i.e. non-techies) might pick up in popularity - especially with newbies - if ready-made examples of installation procedures could be provided, as with this supplied example.   Perhaps it could be useful to reiterate here that reproducibility cannot be guaranteed:-

"Alternatively, the Ungoogled-Chromium Portable Linux compiled binary can be installed.  It has been pre-built.  Note:  We cannot guarantee reproducibility.

_1. Download the Ungoogled-Chromium Portable Linux binary to your preferred location e.g. `~/Downloads`."_

_2. Navigate to the download location.  Check download by comparing md5, sha1 and sha256 sums with those at ungoogled-chromium's download webpage._

_3. Supposing that your chosen installation location is `/opt`, extract the binary, substituting in the name of your download:_

```
$ sudo tar -xvf ungoogled-chromium_xxxxxxx.tar.xz -C /opt
```

Reply:
Note:  This "Installing compiled binary" section has now been moved to __after__ the last section ("Developer info") to keep 'building' discussions together, and the pre-compiled binary discussion afterward.  Please let me know if you prefer anything changed.